crates/agglayer-storage/src/types/certificate/mod.rs

```diff
@@ -101,11 +101,15 @@ impl From<CertificateV0> for Certificate {
             prev_local_exit_root,
             new_local_exit_root,
             bridge_exits,
-            imported_bridge_exits,
+            imported_bridge_exits: imported_bridge_exits.clone(),
             aggchain_data: AggchainData::ECDSA { signature },
             metadata,
             custom_chain_data: vec![],
-            l1_info_tree_leaf_count: None,
+            l1_info_tree_leaf_count: imported_bridge_exits
+                .iter()
+                .map(|i| i.l1_leaf_index() + 1)
+                .max()
+                .unwrap_or(0u32),
```

Comment:
not sure about that one in terms of storage/compatibility @Freyskeyd @iljakuklic

Reply:
There should be similar comaptibility handling for json deserialization too.

* addressed in #769